playwright-tests/utils/confirmPem.ts

```diff
@@ -0,0 +1,16 @@
+import { Page } from '@playwright/test';
+import { DataTestIdsEnum } from '../../src/localConstants/dataTestIds.enum';
+import { LoginFilesEnum } from '../utils/enums.ts';
+
+interface ConfirmPem {
```

Comment:
Too vague naming here, should have a `Type` suffix or something like that indicating it's a type.